WordPress/Resources/ro.lproj/Localizable.strings

```diff
@@ -2682,9 +2682,6 @@ translators: Block name. %s: The localized block name */
 /* Title for the edit sharing buttons section */
 "Edit sharing buttons" = "Editare butoane de partajare";
 
-/* Title for the edit sharing buttons section */
-"Edit sharing buttons" = "Editare butoane de partajare";
-
```

Comment:
Notice this is a duplicated key that made it into GlotPress and hid there for a long time, but our [new duplicated strings tooling](https://github.com/wordpress-mobile/WordPress-iOS/pull/18465) noticed it 😄